src/tests/edje/tests/meson.build

```diff
@@ -6,8 +6,8 @@ foreach edc_file : edc_files
    themes += custom_target('edje_cc_' + edc_file,
     input : edc_file,
     output : '@BASENAME@.edj',
-    command : edje_cc_exe + [ '-beta', '-fastcomp',
-              '-id', join_paths(meson.source_root(), 'src', 'tests', 'emotion', 'data'),
+    command : edje_cc_exe + [ '-beta', '-fastcomp', '-in-tree',
+              '-id', join_paths(meson.source_root(), 'src', 'tests', 'data'),
```

Comment:
?

Reply:
Ok, just a plain "?" isn't a proper way to ask things, so...

Why exactly are we removing `emotion` there?

Reply:
Hmm yeah, that seems wrong to me. I will take a look at all the other places too

Reply:
Maybe this should be at #369 ?

Reply:
actually there is no `src/tests/data` in tree.